src/app/Pages/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { useState } from 'react'
+import Fingerboard from '../../components/Fingerboard/Fingerboard'
+import { fingerboardData } from '../../data/fingerboardData/fingerboardData'
+import Heading from '../../components/Heading/Heading'
+import styled from 'styled-components'
+import Fret from '../../components/Fret/Fret'
+import ArrowButton from '../../components/ArrowButton/ArrowButton'
+
+export default function Dashboard(): JSX.Element {
+  const [pressed, setPressed] = useState(fingerboardData)
+  const [fretOffset, setFretOffset] = useState(0)
+
+  function handleClick(column: number, row: number) {
+    const newPressed = { ...pressed }
+    const pressedFret = ++row + fretOffset
+    if (column === 0) newPressed.e = pressedFret
+    if (column === 1) newPressed.a = pressedFret
+    if (column === 2) newPressed.d = pressedFret
+    if (column === 3) newPressed.g = pressedFret
+    if (column === 4) newPressed.b = pressedFret
+    if (column === 5) newPressed.e2 = pressedFret
+    console.log(newPressed)
```

Comment:
Do you still need this?

Reply:
No i don't, thank you for your attention!